xml/System.Net.Sockets/MulticastOption.xml

```diff
@@ -271,6 +271,8 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="group" /> is <see langword="null" />.</exception>
```

Comment:
If this is specific for .NET 5, then we should add the note prefix:

```suggestion
          .NET 5 and above: <paramref name="group" /> is <see langword="null" />.</exception>
```

@BillWagner should we update documentation for code changes that haven't been released?

Reply:
There's got to be a better way to version the documentation, isn't there?

Reply:
Unfortunately no. AFAIK, we don't have a better way to condition showing an exception for certain versions.

cc @mimisasouvanh @joelmartinez Do we have plans to allow showing exceptions for specific versions/monikers?

Reply:
> should we update documentation for code changes that haven't been released?

We do have a moniker for ".NET 5.0 alpha".  We aren't showing those yet, so we may want to wait to publish until the first public preview goes out.

Reply:
@carlossanlop @joelmartinez @BillWagner We currently don't have plans to support versioning for exceptions. Is there a big need for that?

Reply:
This is only the second time I see us specifying the version that supports an exception. I think we can consider this a low priority for now, but it could be considered for the future.